code/modules/mob/living/simple_animal/damage_procs.dm

```diff
@@ -0,0 +1,37 @@
+
+/mob/living/simple_animal/proc/adjustHealth(amount, updating_health = TRUE)
```

Comment:
Is this going to require all simple_animal adjustHealth procs to be updated?
Has this been done?

Reply:
Yes and yes.